cmd/options.go

```diff
@@ -67,6 +67,7 @@ func optionFlagSet() *pflag.FlagSet {
 	flags.String("summary-time-unit", "", "define the time unit used to display the trend stats. Possible units are: 's', 'ms' and 'us'")
 	flags.StringSlice("system-tags", lib.DefaultSystemTagList, "only include these system tags in metrics")
 	flags.StringSlice("tag", nil, "add a `tag` to be applied to all samples, as `[name]=[value]`")
+	flags.String("redirect-console-to-file", "", "redirects the console logging to the provided file")
```

Comment:
This seems a bit verbose for an option name, though I'm not sure what would be better... Maybe something like `console-output`?